src/sections/dashboard/searchbar/ExpandSearchBar.js

```diff
@@ -375,6 +373,13 @@ export default function ExpandSearchBar({
                 },
               }}
             >
+              {Object.keys(searchResult).every(
```

Comment:
lấy key của searchResult ra là toàn key xong check length của từng cái ở đây thì luôn false vì key luôn có rồi? mà cũng k phải check value nữa nên k bao giờ ra noResult
Cái power search phức tạp nên sẽ phải sửa lại logic chứ hiện giờ work k chuẩn, khó scale

Reply:
values mới đúng em nhầm, để em sửa lại